qiskit/providers/__init__.py

```diff
@@ -500,6 +546,110 @@ def result(self):
         def status(self):
             return JobStatus.DONE
 
+======================================
+Migrating between Backend API Versions
+======================================
+
+BackendV1 -> BackendV2
+======================
+
+The :obj:`~BackendV2` class re-defined user access for most properties of a
+backend to make them work with native Qiskit data structures and have flatter
+access patterns. However this means when using a provider that upgrades
+from :obj:`~BackendV1` to :obj:`~BackendV2` existing access patterns will need
+to be adjusted. It is expected for existing providers to deprecate the old
+access where possible to provide a graceful migration, but eventually users
+will need to adjust code. The biggest change to adapt to in :obj:`~BackendV2` is
+that most of the information accesible about a backend is contained in it's
+:class:`~qiskit.transpiler.Target` object and the backend's attributes often query
+it's :attr:`~qiskit.providers.BackendV2.target`
```

Comment:
its

Reply:
Fixed in the current revision (just forced pushed as part of a rebase too)